javascript/Mini-lesson on destructuring

```diff
@@ -0,0 +1,57 @@
+### Projected Time
+
+About 25-30 minutes
+- 10 min for destructuring
+- 20 min for examples
+
+### Prerequisites
+
+Here are the topic that should be understood before this topic:
+- Javascript 7
+
+### Destructuring
+
+Destructuring simply implies breaking down a complex structure into simpler parts. In JavaScript, this complex structure is usually an 
+object or an array. With the destructuring syntax, you can extract smaller fragments from arrays and objects. Destructuring syntax can be
+used for variable declaration or variable assignment.
+
+### Why it is useful?
+
+There are two big advantages of using destructuring. The first one is that your code more readable. If you destructure an object at the
+top of a function or code block, it is clear to the reader what variables you are going to use.
+The second plus is performance. Destructuring encourages programmers to put object properties in local variables before using them,
+which can improve application performance. Especially if you are accessing those variables multiple times, maybe in a loop, it is more
+efficiënt if the variables are locally defined.
+
+### Examples
+
+### Example 1 ->
+
+Destructuring an two element array into two named variables.
+
+### Code with destructuring :
+
+let { type, color } = car;
```

Comment:
Be sure to indent lines if they are supposed to be under a section. Also, use backticks to denote code.
```suggestion
  `let { type, color } = car;`
```